packages/shared/src/WithNativeNavigation.js

```diff
@@ -9,7 +9,7 @@ type NavigationState = {|
 
 type InjectedProps = {|
   onNavigationStateChange: () => void,
-  onBackClicked: () => void,
+  onBackClicked?: () => void,
```

Comment:
Why is this optional now? What has changed?

Reply:
Reverted. Added `FlowFixMe` till `onBackClicked` will not be implmented in Android app.